### PR TITLE
Update speakers-guide.md

### PR DESCRIPTION
**Pull request by lo-wi**
Added the contents of https://docs.google.com/document/d/1uAOtT6MQYjlKtuJSM6Uz-UPKKk29Ms05rIVVouQghc4/edit?usp=sharing. Tried to keep common parts on top. However, this still needs more structure.

### CODE REVIEW

File: organizing/speakers-guide.md [hunk 1]
```diff
@@ -1,9 +1,62 @@
...
+    * From our experience a self-paced format where attendees take their own time with coaches to support them is best for mixed level audience. 
+    * We	highly recommend to pair people as part of the workshop.
+    * We have also had a lot of success with an interactive format where attendees had to choose cards with workshop-related topics which they then had to explain to each other. Example: https://github.com/PyLadiesBerlin/resources/blob/master/workshops/23_Data_Wrangling_Pandas/README.md
+    * If it is a code along format, consider a catch-up point giving people time to catch up if they are left behind.
+    * If there are exercises to complete, it is generally a useful practice to have already solved shortcut steps that the attendees can take if they get stuck (e.g. folder with solved steps - example: https://github.com/mei-li/multiling-chat)
+
+* **Workshop coaches:** Usually, if the workshop is self-paced, it is important to have people (coaches) helping on the spot. Having a couple of coaches for technical issues is always helpful. Let us know, if coaches are needed.
```
Comment (mei-li) [inline organizing/speakers-guide.md]: maybe here it is worth to say that the audience can be up to 40 people so more coaches might be needed

File: organizing/speakers-guide.md [hunk 2]
```diff
@@ -1,9 +1,62 @@
...
+This is a guide for speakers at Pyladies Berlin meetups. The following guidelines apply to **talks and workshops**; for workshops, you find additional recommendations below.
 
-TODO add the contents of https://docs.google.com/document/d/1uAOtT6MQYjlKtuJSM6Uz-UPKKk29Ms05rIVVouQghc4/edit?usp=sharing
+* **Audience level:** Expect all levels. From people that have never tried Python before to Python data science professionals and engineers. In our meetups, we aim that beginners (having programmed a bit before) are able to follow most of the things. 
 
-keep common parts on top, if it is too much maybe we can reduce the info to useful level
+* **Coaching:** We offer coaching. That means you can discuss your talk/workshop material with one of us and ask to go through the slides/workshop outline and give feeback. It is also possible to arrange a talk rehearsal session live or via video call. Pleasse get in touch with us!
 
-* Talk 
+#Maybe we could add some guidelines here on talk length, language and ideas for talks or talks that have been given in the past. I couldn't find this in the google doc.
```
Comment (mei-li) [inline organizing/speakers-guide.md]: We can encourage a 20-30 mins that is also suitable for a conference later, but anything less is absolutely welcome as well, also lightning talks of 5 mins. The only thing is that we need to know that in timely manner to set the schedule accordingly.

**Comment by mei-li** [comment]
I reopened it. If you want you can address the comments of add a `TODO` infront of the suggestion and we can merge it!

**Comment by lo-wi** [comment]
Oh great - glad that not all work was lost...
Just to make sure I understand: I could try to 1. incorporate the
suggestions in your comments and 2. address all other left-over issues by
adding them with a TODO in front?
I was also thinking: shall we add emojis to make it all look a little more
approachable? Never sure about this.

Am Fr., 2. Aug. 2019 um 10:05 Uhr schrieb Meili Triantafyllidi <
notifications@github.com>:

**Comment by mei-li** [comment]
yes for emojis, it is a great idea!!!!

**Comment by mei-li** [comment]
It looks great :+1: 

I will merge it!

**Pull request merged**